### PR TITLE
Remove udp/rtsp h264

### PR DESCRIPTION
**Pull request by tszumski**
Remove udp/rtsp h264, 
To be merged after: https://github.com/OpenVisualCloud/Media-Communications-Mesh/pull/213

### CODE REVIEW

File: media-proxy/include/proxy_context.h [hunk 1]
```diff
@@ -44,8 +44,6 @@ class ProxyContext {
     libfabric_ctx *mDevHandle_rdma = NULL;
 
     /*udp pool*/
```
Comment (Sakoram) [inline media-proxy/include/proxy_context.h:46]: remove the comment too

File: media-proxy/src/proxy_context.cc [hunk 2]
```diff
@@ -570,7 +570,7 @@ int ProxyContext::RxStart_mtl(const mcm_conn_param *request)
     int ret;
 
     /*add lock to protect MTL library initialization to aviod being called by multi-session simultaneously*/
-    if (!mDevHandle && imtl_init_preparing == false) {
+    if (mDevHandle == NULL && imtl_init_preparing == false) {
```
Comment (Sakoram) [inline media-proxy/src/proxy_context.cc:573]: Why this change? We usually use !val to check if value is NULL.

Comment (tszumski) [reply media-proxy/src/proxy_context.cc:573]: To align with code in TxStart_mtl()

Comment (ko80) [reply media-proxy/src/proxy_context.cc:573]: > To align with code in TxStart_mtl()

Could you elaborate on that?

Comment (Sakoram) [reply media-proxy/src/proxy_context.cc:573]: I guess we just have to define our approach. In a new code we tend to use `!val`, in old code there is a mix of everything.
Let's agree on something. Me personally I'm okay with both approaches. `val == NULL` is slightly more readable, `!val` is smaller and more elegant.

Comment (ko80) [reply media-proxy/src/proxy_context.cc:573]: I vote for `if (!something) {}`.

Comment (tszumski) [reply media-proxy/src/proxy_context.cc:573]: I also vote for ```!val```

Comment (Sakoram) [reply media-proxy/src/proxy_context.cc:573]: I declare to all and sundry that the winner is `!val` :tada:

File: media-proxy/include/proxy_context.h (hunk 1 above)
Comment (tszumski) [reply media-proxy/include/proxy_context.h]: done

File: media-proxy/src/proxy_context.cc (hunk 2 above)
Comment (tszumski) [reply media-proxy/src/proxy_context.cc]: Removed this change

**Review by Sakoram — APPROVED** [review]
LGTM, but before merging, please run our samples from tests/single-node-sample-apps/ to make sure everything is in ok

**Review by ko80 — APPROVED** [review]
LGTM
